index.js

```diff
@@ -45,6 +48,7 @@ class K8sExecutor extends Executor {
 
             this.token = fs.existsSync(tokenPath) ? fs.readFileSync(tokenPath).toString() : '';
         }
+        this.buildTimeout = hoek.reach(options, 'kubernetes.buildTimeout') || DEFAULT_BUILD_TIMEOUT;
```

Comment:
can also write this `hoek.reach(options, 'kubernetes.buildTimeout', { default: DEFAULT_BUILD_TIMEOUT });`
not sure what's better

Reply:
Exceeds the ESLint line width

Reply:
🤦‍♀️

Reply:
IKR 🙄